### ISSUE TITLE
Issues on paper

### ISSUE THREAD

**Issue by hitcbw** [geyanqi/SAR#2]
The code is not "is available at..."

**Comment by geyanqi** [comment]
Hi, thanks for your attention to our work. Currently, I'm in a rush for a conference deadline. We will release the code as soon as the ddl is ended.

**Comment by hitcbw** [comment]
Thanks for your reply. Now i am reproducing the network but the CE reweighting makes me confused.
1. At the early training stage, the prediction tends to random distribution. The weight would be very small when number of classes is large.
2. I suppose that eq.6 & 7 works after softmax. Then, eq.6 set w_c as 0 and it would generate -inf in eq.7. How it works?

Looking forward to your further instruction. Regards.

**Comment by geyanqi** [comment]
for q2:
Thank you very much for noticing the typos in eq.7. In addition, w is indeed the result after softmax, and in eq.7, the denominator should be the sum of the log of those non-0 w values. 
for q1:
In the initial stage of training, the auxiliary cross-entropy loss will not become smaller due to the reweighting strategy. You can think of the number of classes as batch data passed into the cross-entropy. Compared to traditional cross-entropy, the weight of each term is not 1/batch, but the weighted weight in eq.7.

**Comment by hitcbw** [comment]
Thanks for your reply. So much clearer now.

**Comment by geyanqi** [comment]
Hello, we have released the code.
